src/ci/docker/scripts/build-clang.sh

```diff
@@ -34,7 +34,7 @@ hide_output \
       -DCOMPILER_RT_BUILD_XRAY=OFF \
       -DCOMPILER_RT_BUILD_MEMPROF=OFF \
       -DCOMPILER_RT_BUILD_CTX_PROFILE=OFF \
-      -DLLVM_TARGETS_TO_BUILD=X86 \
+      -DLLVM_TARGETS_TO_BUILD=$LLVM_BUILD_TARGETS \
```

Comment:
nit: I would add `u` to the bash script modifier at the top of this script, just to make sure that this env. var isn't forgotten. Otherwise it could be tricky to debug the CMake failure.

Reply:
Fixed